File: Tests/UnitTestImpromptuInterface.VBNet/Support/FixtureSetup.vb
Imports ImpromptuInterface.Build
Imports NUnit.Framework

Namespace VBNET

    <SetUpFixture()>
    Public Class FixtureSetup

        Private Builder As IDisposable

        <SetUp()>
        Public Sub Setup()

            Builder = BuildProxy.WriteOutDll("ImpromptuEmit.VB")

        End Sub

        <TearDown()>
        Public Sub TearDown()

            Builder.Dispose()
        End Sub
    End Class
End Namespace

File: Tests/UnitTestImpromptuInterface/Support/FixtureSetup.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using ImpromptuInterface;
using ImpromptuInterface.Build;


namespace UnitTestImpromptuInterface
{
#if !SILVERLIGHT && !SELFRUNNER && !NETCOREAPP2_0

    using NUnit.Framework;
    [SetUpFixture]
    public class FixtureSetup
    {
        private IDisposable Builder;
        
        [OneTimeSetUp]
        public void Setup()
        {
            Builder = BuildProxy.WriteOutDll("ImpromptuEmit");
        }
        [OneTimeTearDown]
        public void TearDown()
        {
            Builder.Dispose();
        }
    }
#endif
}
